src/version.ts

```diff
@@ -176,6 +203,91 @@ export class Version extends Component {
     this.project.addPackageIgnore(`/${changelogFile}`);
     this.project.addPackageIgnore(`/${bumpFile}`);
   }
+
+  /**
+   * Return the environment variables to set modify the bump command for release branches.
+   *
+   * These options are used to modify the behavior of the version bumping script
+   * for additional branches, by setting environment variables.
+   *
+   * No settings are inherited from the base `Version` object (but any parameters that
+   * control versions do conflict with the use of a `nextVersionCommand`).
+   */
+  public releaseBranchParameters(
```

Comment:
Not crazy about the name here.

Maybe this?

```suggestion
  public envForBranch(
```

Reply:
Done